src/modules/auth/components/Form/LoginForm/LoginForm.jsx

```diff
@@ -0,0 +1,78 @@
+import { useContext, useState } from "react";
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import * as yup from "yup";
+import { Button } from "../../../../../components/Button/Button";
+import { Input } from "../../../../../components/Input/Input";
+import AuthContext from "../../../../../context/auth/auth-context";
+import styles from "../form.module.scss";
+import { DUMMY_LOGINS } from "../../../../../mock/auth";
+
+export const LoginForm = () => {
+  const authContext = useContext(AuthContext);
+  const [validData, setValidData] = useState(true);
+
+  const schema = yup.object().shape({
+    login: yup
+      .string()
+      .max(25, "Maksymalnie 25 znaków")
+      .required("Podaj poprawny email/login"),
+    password: yup
+      .string()
+      .min(4, "Minimum 4 znaki")
+      .max(15, "Maksymalnie 15 znaków")
+      .required("Podaj hasło"),
+  });
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm({ resolver: yupResolver(schema) });
+
+  const login = () => {
+    authContext.onLogin();
```

Comment:
You can destructure the context like `const { isLoggedIn , onLogin } = useContext(AuthContext);`